ansible/roles/ocp-client-vm/defaults/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+non_root_os_user: "ec2-user"
```

Comment:
there is another variable `remote_user` and `ansible_ssh_user` don't you want to use that instead ?

Reply:
@jbride @fridim Agreed. Any reason we couldn't use `remote_user`